hatServer.py

```diff
@@ -0,0 +1,110 @@
+from flask import Flask, render_template, request
```

Comment:
Hayden's thing should be merged tonight, then we can remove this file and the sorting file